src/trackers/flood.py

```
@@ -0,0 +1,281 @@
+        headers = {"User-Agent": f"Upload Assistant/2.2 ({platform.system()} {platform.release()})", "Authorization": f"Bearer {self.api_key}"}
+
+        if not meta.debug:
+            async with httpx.AsyncClient() as client:
+                response = await client.post(url=self.upload_url, files=files, data=data, headers=headers)
+            try:
+                response.raise_for_status()
+                response_data = response.json()
+            except httpx.HTTPStatusError as e:
+                logger.info(f"{self.tracker}: [red]Upload failed (HTTP Error): {e}")
+                return False
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)